python/metatensor-operations/metatensor/operations/block_from_array.py

```diff
@@ -49,22 +59,41 @@ def block_from_array(array) -> TensorBlock:
             must have at least two dimensions. Too few provided: {n_dimensions}"
         )
 
+    if torch_jit_is_scripting():
+        # for metatensor-torch the array backend is always torch
+        labels_array_like = torch.empty(0)
+    else:
+        # This will return true only if we are using metatensor-torch
+        # please refer to function definition in files
+        # metatensor-core python/metatensor-operations/metatensor/operations/_classes.py
+        # metatensor-torch python/metatensor-torch/metatensor/torch/operations.py
+        if check_isinstance(None, Labels):
```

Comment:
I don't love this, it feels very hacky. Could this do a `if isinstance(Labels, torch.ScriptClass)` instead?

Reply:
sure